<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="203722" started-at="2014-01-15T09:41:40Z" finished-at="2014-01-15T09:45:04Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="203722" started-at="2014-01-15T09:41:40Z" finished-at="2014-01-15T09:45:04Z">
      <class name="hub.test.UserCartCountRestrictionTest">
        <test-method status="PASS" signature="Startup(java.lang.String)[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="Startup" is-config="true" duration-ms="25969" started-at="2014-01-15T17:41:40Z" finished-at="2014-01-15T17:42:06Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Startup -->
        <test-method status="FAIL" signature="testCartCountRestriction(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="testCartCountRestriction" duration-ms="87965" started-at="2014-01-15T17:42:06Z" description="Cart Count Restriction" data-provider="Data-Provider-Function" finished-at="2014-01-15T17:43:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@2028aa2f]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.UserCartCountRestrictionTest.testCartCountRestriction(UserCartCountRestrictionTest.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartCountRestriction -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="reinitializeBrowser" is-config="true" duration-ms="2667" started-at="2014-01-15T17:43:34Z" finished-at="2014-01-15T17:43:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testCartCountRestriction(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="testCartCountRestriction" duration-ms="85115" started-at="2014-01-15T17:43:37Z" description="Cart Count Restriction" data-provider="Data-Provider-Function" finished-at="2014-01-15T17:45:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@4dbd0427]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.UserCartCountRestrictionTest.testCartCountRestriction(UserCartCountRestrictionTest.java:66)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCartCountRestriction -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="reinitializeBrowser" is-config="true" duration-ms="1507" started-at="2014-01-15T17:45:02Z" finished-at="2014-01-15T17:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:hub.test.UserCartCountRestrictionTest@3a1cd84e]" name="teardown" is-config="true" duration-ms="156" started-at="2014-01-15T17:45:03Z" finished-at="2014-01-15T17:45:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- hub.test.UserCartCountRestrictionTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
